.gitignore

```diff
@@ -8,3 +8,4 @@ art-contrasts.html
 art-effect-size.html
 cache
 .RData
```

Comment:
You should add .DS_Store to .gitignore and remove the .DS_Store files MacOS annoying sticks everywhere from the repo :)

Reply:
Also remove R/.Rapp.history